addons/stock/static/src/js/stock_traceability_report_backend.js

```diff
@@ -39,7 +40,7 @@ var stock_report_generic = Widget.extend(ControlPanelMixin, {
         def.then(function () {
             self.report_widget.$el.html(self.html);
             if(self.given_context['ttype'] == 'downstream'){
-                self.report_widget.$el.find('.o_report_heading').html('<h1>Downstream Traceability</h1>');
+                self.report_widget.$el.find('.o_report_heading').html(_t('<h1>Downstream Traceability</h1>'));
```

Comment:
Careful, you open the door to a potential XSS injection here (if translated by `<script>...</script>`). If the content become dynamic, it should be added via `.text()` and not `.html()`, so need to adapt a bit the js code too.

Reply:
Hmm interesting, I didn't know that actually! 🤔 
Would this work then? Or would it still keep a backdoor open because it is sent through the network?
```suggestion
                var report_title = _t('<h1>Downstream Traceability</h1>');
                self.report_widget.$el.find('.o_report_heading').html(report_title);
```

`.text()` does not allow HTML so I'm not sure what you mean with the "so need to adapt a bit the js code too" honestly.

Reply:
You don't want to have dynamic content injected in a `.html(...)`. The content must be injected with a `.text(...)`. Something like this (untested):

```suggestion
                var heading = $("<h1></h1>").text(_t("Downstream Traceability"));
                self.report_widget.$el.find('.o_report_heading').empty.append(heading);
```